snapcraft.yaml

```diff
@@ -0,0 +1,19 @@
+name: tqdm
+version: '1.0'
+summary: A fast, extensible progress bar for Python and CLI
+description: |
+  A fast, extensible progress bar for Python and CLI
+grade: stable
+confinement: strict
+base: core18
+icon: tqdm-logo.png
+parts:
+  tqdm:
+    plugin: python
+    python-version: python2
```

Comment:
python3 is also supported

Reply:
Yes, absolutely, you can also build with python3. Unless you use specific python3 stuff, then it's best to build with python2 for max. compatibility.

Reply:
due to #359 might be best to have python3 only

Reply:
Sure thing, so you simply declare python-version: python3 instead.